examples/scala_rockstar.py

```diff
@@ -0,0 +1,5 @@
+from RockStar import RockStar
+
+scala_code = "object HelloWorld {def main(args: Array[String]) { println(\"Hello world\") }}"
```

Comment:
A valid string literal in scala has to be enclosed in double quotes else code won't compile. I wanted to have the hello world generated by rockstar script in correct syntax.